core/jvm/test/JvmPlatformTest.kt

```diff
@@ -0,0 +1,135 @@
+/*
+ * Copyright 2010-2023 JetBrains s.r.o. and Kotlin Programming Language contributors.
+ * Use of this source code is governed by the Apache 2.0 license that can be found in the LICENSE.txt file.
+ */
+
+/*
+ * Copyright (C) 2018 Square, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package kotlinx.io
+
+import org.junit.jupiter.api.io.TempDir
+import java.io.ByteArrayInputStream
+import java.io.ByteArrayOutputStream
+import java.io.File
+import java.net.Socket
+import java.nio.file.StandardOpenOption
+import kotlin.test.*
+
+class JvmPlatformTest {
+    @TempDir
+    lateinit var tempDir: File
+
+    @Test fun outputStreamSink() {
+        val baos = ByteArrayOutputStream()
+        val sink = baos.sink()
+        sink.write(Buffer().writeUtf8("a"), 1L)
+        assertArrayEquals(baos.toByteArray(), byteArrayOf(0x61))
+    }
+
+    @Test fun inputStreamSource() {
+        val bais = ByteArrayInputStream(byteArrayOf(0x61))
+        val source = bais.source()
+        val buffer = Buffer()
+        source.read(buffer, 1)
+        assertEquals(buffer.readUtf8(), "a")
+    }
+
+    @Test fun fileSink() {
```

Comment:
Would it be okay for you to auto-format all the tests in a follow-up PR?

We typically have slightly different code-styles in `kotlinx-` libraries, and it's okay (and it's quite pointless to establish *the* way across a bunch of independent libraries), but it would be way too easy to confuse
`@Test foo()` vs `@Test\nfoo()` and switch the context all the time. We prefer the latter

Reply:
Yes, I'll definitely be okay